src/app/components/StartScreen.tsx

```diff
@@ -32,40 +57,64 @@ function StartScreen() {
 
   return (
     <div className="h-auto p-4 my-auto content scroll-container">
-      <Stack direction="column" gap={4}>
-        <a href="https://jansix.at/resources/figma-tokens?ref=figma-tokens-plugin" target="_blank" rel="noreferrer">
-          {/* eslint-disable-next-line */}
-          <img alt="Figma Tokens Splashscreen" src={require('../assets/tokens-intro.jpg')} className="rounded" />
-        </a>
+      <Stack direction="column" gap={4} css={{ padding: '2rem', background: '$startScreenBg' }}>
```

Comment:
```suggestion
      <Stack direction="column" gap={4} css={{ padding: '2rem', background: '$bgSubtle', justifyContent: 'center' }}>
```

This is what it looked like for me in Light mode. 
![image](https://user-images.githubusercontent.com/4548309/194640927-1d2ce7f6-03ae-4787-80eb-6f4c36d29b40.png)